apps/src/studio/studio.js

```diff
@@ -418,6 +426,24 @@ var drawMap = function () {
   score.setAttribute('visibility', 'hidden');
   svg.appendChild(score);
 
+  var victoryText = document.createElementNS(SVG_NS, 'text');
+  victoryText.setAttribute('id', 'victoryText');
+  victoryText.setAttribute('class', 'studio-victory-text');
+  victoryText.setAttribute('x', Studio.MAZE_WIDTH / 2);
+  victoryText.setAttribute('y', VICTORY_TEXT_Y_POSITION);
+  victoryText.appendChild(document.createTextNode(''));
+  victoryText.setAttribute('visibility', 'hidden');
+  svg.appendChild(victoryText);
+
+  var resetText = document.createElementNS(SVG_NS, 'text');
+  resetText.setAttribute('id', 'resetText');
+  resetText.setAttribute('class', 'studio-reset-text');
+  resetText.setAttribute('x', Studio.MAZE_WIDTH / 2);
+  resetText.setAttribute('y', RESET_TEXT_Y_POSITION);
+  resetText.appendChild(document.createTextNode(''));
+  resetText.setAttribute('visibility', 'hidden');
+  svg.appendChild(resetText);
+
```

Comment:
We should consider whether to place these in a layer/group to ensure predictable z-order.